app/routes/resume.tsx

```
@@ -0,0 +1,92 @@
+                    )}
+                </section>
+                <section className="feedback-section">
+                    <h2 className="text-4xl !text-black font-bold">Resume Review</h2>
+                    {feedback ? (
+                        <div className="flex flex-col gap-8">
+                            <Summary feedback={feedback} />
+                            <ATS score={feedback.ATS.score || 0} suggestions={feedback.ATS.tips || []} />
+                            <Details feedback={feedback} />
+                        </div>
+                    ) : (
+                        <img src="/images/resume-scan-2.gif" className="w-full" />
```

Biome (2.1.2)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)